site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -287,6 +287,8 @@ The following tables lists the configurable parameters of the Agones chart and t
 | Parameter                                                | Description                                                                                                                                                                                                                                                                                                      | Default                                   |
 |----------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|
 | `agones.extensions.http.port`                            | Port to use for liveness probe service and metrics                                                                                                                                                                                                                                                               | `8080`                                    |
+|`agones.extensions.webhooks.port`                         | Port to use for webhook service                                                                                                                                                                                                                                                                                  | `8081`                                    |
+|`agones.extensions.hostNetwork`                           | Determines if the Agones extensions should operate in hostNetwork mode. If running in hostNetwork mode, you should change `http.port` and `webhooks.port` to an available port.                                                                                                                                  | `false`                                   |
```

Comment:
Two things.

(non blocking question): Should `agones.extensions.hostNetwork` go first, since it refers to `agones.extensions.webhooks.port` ? 

Same as above, will need a feature shortcode. The renderer tends to dislike it in a middle of a table, so you will need to duplciate the table and have one that has a `publishVersion` and one with an `expireyVersion`.